includes/class-wc-admin-reports-segmenting.php

```diff
@@ -311,6 +311,7 @@ protected function set_all_segments() {
 				'limit'  => -1,
 			);
 
+			// @todo: filter by categories if $this->query_args['categories'] is set.
 			if ( isset( $this->query_args['product_includes'] ) ) {
```

Comment:
When segmenting by products and filtering by `categories`, all products are returned with zero'ed values, which results in oddness in the chart seen below (Sunglasses are not in the Hoodies category)

<img width="704" alt="screen shot 2019-03-07 at 6 21 39 pm" src="https://user-images.githubusercontent.com/1922453/53934073-18037600-4106-11e9-8267-fcf55abd9916.png">

I'd like to address this in a follow up PR.